### ISSUE TITLE
Missing documentation on crates.io

### ISSUE THREAD

**Issue by realcr** [jedisct1/rust-jwt-simple#29]
Hi, thanks for writing this crate!
The project's documentation is missing from the crates.io listing. A screenshot is attached:
![Screenshot from 2021-06-17 12-04-21](https://user-images.githubusercontent.com/5255681/122366373-2cc94800-cf64-11eb-8ae6-fdf9f664099a.png)

**Comment by jedisct1** [comment]
Click on `jwt-simple`?

There's even a `Documentation` link, then.

**Comment by realcr** [comment]
You are right, sorry! Somehow I couldn't see it on the first time I checked.
I wonder, how come some crates (for example in the picture above: jwt) has a documentation link right from the search page, but some don't?
